shell/creators/extension/app/init

```diff
@@ -89,7 +95,11 @@ Object.keys(targets).forEach((target) => {
 const creatorPkgData = fs.readFileSync(path.join(__dirname, 'package.json'));
 const creatorPkg = JSON.parse(creatorPkgData);
 
-pkg.dependencies['@rancher/shell'] = `^${ creatorPkg.version }`;
+// Fetch the latest version of @rancher/shell
+const shellVersion = execSync('npm view @rancher/shell version').toString().trim();
```

Comment:
Do we want to do this?

Previous behaviour is that the creator will create etc using the shell version of the creator.

For the update behaviour, the idea being that the logic to do the update was in the creator with the same version as the shell.

Reply:
How does a user create a new extension with a specific shell version?